src/lib/rpc.ts

```diff
@@ -102,11 +103,18 @@ export const makeRpcUtils = async ({ agoricNet }) => {
       ),
   };
 
+  const storageWatcher = makeAgoricChainStorageWatcher(
+    rpcAddrs[Math.floor(Math.random() * rpcAddrs.length)],
```

Comment:
https://lodash.com/docs/4.17.15#sample would convey this well.
```suggestion
    _.sample(rpcAddrs),
```

if we have tree shaking then adopting https://www.npmjs.com/package/lodash-es would have only a small bundle weight

Reply:
Done. It added 7KB uncompressed / 2KB compressed, not terrible I guess.